### PR TITLE
[O2B-1303] Use icon for missing verification display

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- Replaced previous display for missing verifications with warning icon adding tooltip to it
- Changed display of Add QC button in QC summary to take all available space

Notable changes for developers:
- NA

Changes made to the database:
- NA

### CODE REVIEW

File: test/public/runs/runsPerSimulationPass.overview.test.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ module.exports = () => {
         await reloadPage(page);
         await expectLink(page, 'tr#row56 .column-ITS a', {
             href: 'http://localhost:4000/?page=qc-flags-for-simulation-pass&runNumber=56&dplDetectorId=4&simulationPassId=2',
-            innerText: '0!',
+            innerText: '0',
```
Comment (martinboulais) [inline test/public/runs/runsPerSimulationPass.overview.test.js:131]: You should probably still look for the icon to be there

File: lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js [hunk 2]
```diff
@@ -42,8 +42,13 @@ const runDetectorAsyncQualityDisplay = ({ dataPassId, simulationPassId }, run, d
     } = qcSummary[runNumber][dplDetectorId];
 
     const notBadPercentageFormat = [
-        formatFloat((1 - badEffectiveRunCoverage) * 100, { precision: 0 }),
-        missingVerificationsCount ? h('sub', '!') : null,
+        h('span', formatFloat((1 - badEffectiveRunCoverage) * 100, { precision: 0 })),
+        h('.d-inline-block', missingVerificationsCount
+            ? tooltip(
+                h('sub', iconWarning()),
```
Comment (martinboulais) [inline lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js:48]: no sub

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js (hunk 2 above)
Comment (xsalonx) [reply lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js]: applied

File: test/public/runs/runsPerSimulationPass.overview.test.js (hunk 1 above)
Comment (xsalonx) [reply test/public/runs/runsPerSimulationPass.overview.test.js:131]: done

File: lib/public/components/qcFlags/qcFlagsPagesButtons.js [hunk 3]
```diff
@@ -69,7 +70,7 @@ export const qcFlagCreationPanelLink = (
                             pageName,
                             { runNumber: run.runNumber, dplDetectorId, ...options },
                             {
-                                class: 'btn btn-primary',
+                                class: `btn btn-primary ${linkClasses.join(' ')}`,
```
Comment (martinboulais) [inline lib/public/components/qcFlags/qcFlagsPagesButtons.js:73]:
```suggestion
                                class: ['btn', 'btn-primary', ...linkClasses].join(' ')}`,
```

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/components/qcFlags/qcFlagsPagesButtons.js (hunk 3 above)
Comment (xsalonx) [reply lib/public/components/qcFlags/qcFlagsPagesButtons.js]: applied

**Review by martinboulais — APPROVED** [review]

**Pull request merged**